src/Components/App/App.js

```diff
@@ -3,31 +3,36 @@ import LandingPage from '../LandingPage/LandingPage';
 import Footer from '../Footer/Footer';
 import OutboundLink from '../OutboundLink/OutboundLink';
 import TherapistPage from '../TherapistPage/TherapistPage';
-import { Switch, Route } from 'react-router-dom';
+import ErrorMessage from '../ErrorMessage/ErrorMessage';
+import BadUrl from '../BadUrl/BadUrl';
+import { Switch, Route, Redirect } from 'react-router-dom';
 import { useTherapists } from '../../utilities';
 
 const App = () => {
   const { data, loading, error } = useTherapists();
 
   if (loading && !data) return "Loading...";
-
-  if (error) return `${error.message}`;
-
+  
   return (
     <main className="app-main">
       <header className="app-header">
         <h1 >Equilibrium</h1>
       </header>
-      <Switch >
+      {error
+      ? <ErrorMessage error={error} />
```

Comment:
It looks like this is intended to be a full-page error - I suggest making an error route and directing the user there if there is an error. This way it is scalable if any other API requests are added, you can send them to the error page.

You can utilize `useEffect` to listen for the `error` value to change, and redirect the user when it does. For example --
```
const navigate = useNavigate();
useEffect(() => {
  if(error) {
    navigate("/error", { state: { error }});
  }
});
```